### ISSUE TITLE
Simple JSON breaking in @next

### ISSUE THREAD

**Issue by natew** [typeorm/typeorm#1898]
Logging this in it's own ticket.

**Issue type:**

[ ] question
[x] bug report
[ ] feature request
[ ] documentation issue

**Database system/driver:**

[ ] `cordova`
[ ] `mongodb`
[ ] `mssql`
[ ] `mysql` / `mariadb`
[ ] `oracle`
[ ] `postgres`
[x] `sqlite`
[ ] `sqljs`
[ ] `websql`

**TypeORM version:**

[ ] `latest`
[x] `@next`
[ ] `0.x.x` (or put your version here)

**Steps to reproduce or a small repository showing the problem:**

```js
import * as T from 'typeorm'

@T.Entity()
export class Setting extends T.BaseEntity {
  @T.PrimaryGeneratedColumn() id: number
  @T.Column() type: string
  @T.Column({ nullable: true })
  token: string
  @T.Column('simple-json', { default: '{}' })
  values: Object
  @T.CreateDateColumn() createdAt: Date
  @T.UpdateDateColumn() updatedAt: Date
}
```

Getting an error here when saving some data and accessing it:

![image](https://user-images.githubusercontent.com/12100/38320873-a52ddd34-37ea-11e8-90ab-e8ded558d56d.png)

Basically it seems to be getting the actual object here, when it expected a string.

**Comment by pleerock** [comment]
would be great if you'll create a PR with a failing test so you or we can fix it quickly.

**Comment by AlexMesser** [comment]
fixed and released in `typeorm@0.2.0-alpha.44`. You can try it via `npm i typeorm@next`.
